toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+ROOT_FOLDER=$(git rev-parse --show-toplevel)
+
+usage() {
+    echo "./setuplkgtoolchain.sh"
+    echo " Syncs toolchain manifests to match LKG build"
+    exit 1
+}
+
+getlkg() {
+    wget -O lkg-3.0-dev.json -nv https://mariner3dailydevrepo.blob.core.windows.net/lkg/lkg-3.0-dev.json
+    DAILY_BUILD_ID=$(cat lkg-3.0-dev.json | jq -r .dailybuildid | tr . -)
+    GIT_COMMIT=$(cat lkg-3.0-dev.json | jq -r .commit)
```

Comment:
Nit-pick: you can have `jq` work on the file directly instead of using `cat`.
```suggestion
    DAILY_BUILD_ID=$(jq -r .dailybuildid lkg-3.0-dev.json | tr . -)
    GIT_COMMIT=$(jq -r .commit lkg-3.0-dev.json)
```

Reply:
fixed